.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  ![Welcome Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:8ff47a85-7250-4d86-8e48-2f346b48b2c1:BannerWelcome.jpg/full/750,/0/default.jpg)
+  (Image: CC-BY license, The Turing Way Community, & Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332808)
+  
+  :tada: Welcome to the 2020 OHBM Hackathon! :tada:
+  We're really excited to have your input on this repository and welcome your idea! :sparkling_heart:
+  If you haven't done so already, please make sure you check out our [README file](https://github.com/ohbm/hackathon2020/blob/master/README.md) and [Code of Conduct](https://github.com/ohbm/hackathon2020/blob/master/code_of_conduct.md).
+
```

Comment:
Maybe add something like: "Did you issue a new hackathon project? Please check if you have provided all relevant [information] (LINK?) and an image for the [website](https://ohbm.github.io/hackathon2020/hackathon/)."
(if desirable)

Reply:
Hey Dorian, You can directly edit the file by using the '+-' symbol. I would leave the editing totally up to you and other folks from the organisation. 🎉  
![image](https://user-images.githubusercontent.com/5370471/83887703-b97ddb00-a740-11ea-8aed-9e78899e036e.png)

Do call me in if there is some error that need fixing.